css/index.css

```diff
@@ -143,4 +143,158 @@ footer nav {
 footer nav a {
     color: white;
     text-decoration: none;
+}
+
+/* sprint challenge
+
+html {
+    font-size:62.5%;
+}
+*/
+
+
+a{
+    text-decoration: none;
+    color: black;
+}
+
+/* Desktops and laptops ----------- */
+@media only screen  and (min-width : 1000px) and (max-width : 2000px) {
+    /* Styles */
+  
+
+   nav {
+       display: flex;
+       justify-content: space-between;
+       padding: 20px 10px;
+   }
+
+   ul {
+       display: flex;       
+   }
+
+   .middle-content .boxes {
+       width: 920px;
+   }
+
+   li {
+       margin: 0 40px 0 40px;
+       padding-top: 20px;
+       
+   }
+
+   .top-content {
+       padding-top: 20px;
+   }
+   
+    .container ul {
+     display: flex;     
+     margin: 0 auto;
+ }
+
+    .middle-content .boxes .box:nth-of-type(2) {
+    background-color: forestgreen;       
```

Comment:
Watch the indentations here. Also, you can save yourself some typing by using `background: color`